packages/formdescriptioneditors/frontend/sirius-components-formdescriptioneditors/src/SplitButtonWidget.tsx

```diff
@@ -253,11 +254,11 @@ export const SplitButtonWidget = ({ widget, editingContextId, representationId }
   };
 
   const getStyle = (index: number): React.CSSProperties => {
-    if (widget.actions.length == 0) {
+    if (!widget.actions.at(index) || !widget.actions.at(index).style) {
       const defaultProps = {
         backgroundColor: null,
         foregroundColor: null,
-        fontSize: null,
+        fontSize: theme.spacing(1.75),
```

Comment:
Could you use something like `theme.typography.h4.fontSize` instead please? `h4` is an example here, you should check in `sirius-components-core/src/theme.ts` which typography to use.

Reply:
Could you also change it in `helpOutlineOutlined` style please?

Reply:
done